docs/Configuring-Pantheon/Networking/Managing-Peers.md

```diff
@@ -0,0 +1,83 @@
+description: Managing Pantheon peers 
+<!--- END of page meta data -->
+
+# Managing Peers 
+ 
+## Limiting Peers
+
+Limiting peers reduces the bandwidth used by Pantheon. It also reduces the CPU time and disk access 
+used to manage and respond to peers.  
+ 
+Use the [`--max-peers`](../../Reference/Pantheon-CLI-Syntax.md#max-peers) command line option to reduce 
+the maximum number of peers. The default is 25.
+
+## No Discovery
+
+The [`--discovery-enabled`](../../Reference/Pantheon-CLI-Syntax.md#discovery-enabled) command line option 
+can be used to disable P2P peer discovery.
+Set this option to `false` if you are running a test node or a network with [static nodes](#static-nodes).
+
+## Static Nodes
+
+Static nodes are configured nodes that remain connected rather than nodes connected through P2P discovery. 
+Static nodes attempt to maintain connectivity. If a connection goes down to a static node, 
+the node attempts to reconnect every 60 seconds.
+
+To configure a network of static nodes: 
+
+1. List [enode URLs](../Node-Keys.md#enode-url) of the nodes in the [`static-nodes.json` file](#static-nodesjson-file).
+
+1. Save the `static-nodes.json` file in the data directory of each node. 
+
+1. Start Pantheon with discovery disabled using [`--discovery-enabled=false`](../../Reference/Pantheon-CLI-Syntax.md#discovery-enabled).   
+
+To update the static peers, use the [`admin_addPeer`](../../Reference/JSON-RPC-API-Methods.md#admin_addpeer) 
```

Comment:
Maybe "To Modify the static peers at run time...."
''' Note-  runtime modifications of static peers are not persisted between runs

[yeah, being a pedant]

Reply:
Updated